app/controllers/admin/ticket_scannings_controller.rb

```diff
@@ -8,9 +8,14 @@ class TicketScanningsController < Admin::BaseController
     def create
       @ticket_scanning = TicketScanning.new(physical_ticket: @physical_ticket)
       authorize! :create, @ticket_scanning
-      @ticket_scanning.save
-      redirect_to conferences_path,
-                  notice: "Ticket with token #{@physical_ticket.token} successfully scanned."
+      if @physical_ticket.ticket.registration_ticket?
```

Comment:
what about doing this in a before create filter in TicketScanning?

Reply:
👍